ui/src/api/index.ts

```diff
@@ -0,0 +1,39 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
```

Comment:
This file is not generated, remove comments here

Reply:
ok